hivemind/p2p/p2p_daemon.py

```diff
@@ -151,6 +154,7 @@ async def create(
             relayDiscovery=use_relay_discovery,
             autoRelay=use_auto_relay,
             relayHopLimit=relay_hop_limit,
+            idleTimeout=str(idle_timeout) + "s",
```

Comment:
```suggestion
            idleTimeout=f"{idle_timeout}s",
```

nit: We usually prefer f-strings over concatenation in our code base.